src/components/BarChart/Chart.tsx

```diff
@@ -19,11 +19,23 @@ type Props = {
   height: number
   data: ChartData[]
   onFocus: (data: ChartData | null) => void
+  showDays?: boolean
+  upColor?: string
+  downColor?: string
 }
 
-const BarChart = ({ width, height, data, onFocus }: Props) => {
+const BarChart = ({
+  width,
+  height,
+  data,
+  onFocus,
+  showDays = true,
+  upColor,
+  downColor,
+}: Props) => {
   const [focusedBar, setFocusedBar] = useState<ChartData | null>(null)
   const theme = useTheme<Theme>()
```

Comment:
Just wanted to point out that there is a `useColors` hook that simplifies this a little. Either way is fine though.

```
import { useColors } from '../theme/themeHooks'
...
const { grayLight } = useColors()
```